src/Microsoft.Extensions.Caching.Abstractions/IMemoryCache.cs

```diff
@@ -30,5 +30,11 @@ public interface IMemoryCache : IDisposable
         /// </summary>
         /// <param name="key">An object identifying the entry.</param>
         void Remove(object key);
+
+        /// <summary>
+        /// Attempt to remove the given percentage of the total entries in the cache.
+        /// </summary>
+        /// <param name="percentage">The percentage of cache entries to remove.</param>
+        void Compact(double percentage);
```

Comment:
I'm pretty certain we don't want a Count property on the cache - it's utterly meaningless.